unattended_installer/wazuh_install.sh

```diff
@@ -338,19 +338,17 @@ function main() {
 
     logger "Starting Wazuh unattended installer. Wazuh version: ${wazuh_version}. Wazuh installer version: ${wazuh_install_vesion}"
 
-# -------------- Uninstall case  ------------------------------------
+# -------------- Prerequisites and Wazuh repo  ----------------------
 
-    checkIfInstalled
-    if [ -n "${uninstall}" ]; then
-        logger "------------------------------------ Uninstall ------------------------------------"
-        logger "Removing all installed components."
-        rollBack
-        logger "All components removed."
-        exit 0
+    if [ -n "${AIO}" ] || [ -n "${elasticsearch}" ] || [ -n "${kibana}" ] || [ -n "${wazuh}" ]; then
+        logger "---------------------------------- Dependencies -----------------------------------"
+        installPrerequisites
+        addWazuhrepo
     fi
 
 # -------------- Preliminary checks  --------------------------------
 
+    checkTools
```

Comment:
Move this call just after checks import.

Reply:
Done